java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Update exception message in findSyncedMandatoryChannels
```

Comment:
Why is the bsc missing in this entry?

Reply:
It's intentional, the actual fix is only in `spacewalk-web`

Reply:
Thanks for the clarification :-)